src/server/StudioManagementSystem.Core/Entities/GroupApplicationConfig.cs

```diff
@@ -22,8 +22,7 @@ public void Configure(EntityTypeBuilder<GroupApplication> builder)
 
         builder
             .HasOne(e => e.StudentContact)
-            .WithOne(e => e.GroupApplication)
```

Comment:
this was incorrect, I meant to configure a one-to-many for this middle table, but what can you do